back-end/src/main/java/dota/buff/service/impl/MatchServiceImpl.java

```diff
@@ -39,16 +43,19 @@ public MatchDTO getMatchById(long matchId) throws DotaSparkerException {
 
     @Override
     public List<PlayerDTO> getAllMatchPlayers(MatchDTO match) {
+        log.info("Getting all players in match with id: " + match.getMatchId());
         return match.getPlayerList();
     }
 
     @Override
     public List<HeroDTO> getAllMatchHeroes(MatchDTO match) {
+        log.info("Getting all heroes in match with id: " + match.getMatchId());
```

Comment:
Is only id we'd like to see in logs, any more info ab match?

Reply:
No, because the id is unique information that can help in future to fix any bug

Reply:
Please, show me example to make more information in log

Reply:
log.info("Getting all heroes from the match {}",  match);

Reply:
Ok